{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2019-10868",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://discuss.tryton.org/t/security-release-for-issue8189/1262",
          "name": "https://discuss.tryton.org/t/security-release-for-issue8189/1262",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://hg.tryton.org/trytond/rev/f58bbfe0aefb",
          "name": "https://hg.tryton.org/trytond/rev/f58bbfe0aefb",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://seclists.org/bugtraq/2019/Apr/14",
          "name": "20190407 [SECURITY] [DSA 4426-1] tryton-server security update",
          "refsource": "BUGTRAQ",
          "tags": [

          ]
        },
        {
          "url": "https://www.debian.org/security/2019/dsa-4426",
          "name": "DSA-4426",
          "refsource": "DEBIAN",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "In trytond/model/modelstorage.py in Tryton 4.2 before 4.2.21, 4.4 before 4.4.19, 4.6 before 4.6.14, 4.8 before 4.8.10, and 5.0 before 5.0.6, an authenticated user can order records based on a field for which he has no access right. This may allow the user to guess values."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2019-04-05T01:29Z",
  "lastModifiedDate": "2019-04-08T09:29Z"
}